### ISSUE TITLE
FitBounds does not work, if map view is not already set

### ISSUE THREAD

**Issue by bcalik** [SINTEF-9012/PruneCluster#111]
Hello,

I get this error when I call FitBounds and map's view is not predefined or set yet.
I have to call `map.setView(latlng)` before using fitBounds, but what I actually want is to set and fit map view and zoom to marker bounds.

![screen shot 2016-02-08 at 12 27 14](https://cloud.githubusercontent.com/assets/9222368/12883283/919aac1c-ce5f-11e5-8e88-e94b5e737e68.png)

**Comment by bcalik** [comment]
Btw, it works well when I do the work myself like this:

```javascript
var bounds = pruneCluster.Cluster.ComputeGlobalBounds();
if (bounds) {
    map.fitBounds(new L.LatLngBounds(new L.LatLng(bounds.minLat, bounds.maxLng), new L.LatLng(bounds.maxLat, bounds.minLng)));
}
```

**Comment by yellowiscool** [comment]
Hei,

It looks like `this._map` is undefined. It is defined when you add the PruneCluster layer to your map. Do you call `FitBounds` before adding the layer ?

**Comment by bcalik** [comment]
No, PruneCluster layer is already added. I will prepare a html file if you can not reproduce.

**Comment by yellowiscool** [comment]
Yes, that will be helpful.
